scripts/sponsoring/1_add-modules-to-proxy.ts

```diff
@@ -48,7 +55,12 @@ import {
     );
     // Salt but must be the same for schedule & execute
     const operationSalt = '0x0be814a62c44af32241a2c964e5680d1b25c783473c6e7875cbc8071770d7ff0'; // Random
```

Comment:
Why not use an incremental salt `0x0...01`?

Reply:
You are right, see [Use incremental timelock salt](https://github.com/iExecBlockchainComputing/PoCo/pull/132/commits/a1cec05ea4b09f2be3d6a4f3e956c574c242c11a), thanks :pray: